Monika After Story/game/script-topics.rpy

```diff
@@ -13046,3 +13046,149 @@ label mas_show_unseen:
     m 1esa "Just give me a second.{w=0.5}.{w=0.5}.{nw}"
     m 3hua "There you go!"
     return
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_installed_python",
+            category=['misc'],
+            prompt="I installed Python",
+            pool=True,
+            unlocked=True
+        )
+    )
+
+label monika_installed_python:
+    m 1sub "You did?"
+    m 1hub "That's great, [player]!"
+    m 1eub "What did you download it for?{nw}"
+    $ _history_list.pop()
+    menu:
+        m 1eub "What did you download it for?{fast}"
+        "A personal project.":
+            m 1hub "How fun!"
+            m 3hub "Good luck with whatever you're programming!"
+            m 1eub "I'll teach you as much as I can!"
+            m 3ekbfa "Once you learn the basics, could you help me out with a few things?"
+            m 1hub "I know you'll be even better than me one day, and you could even teach me more about programming at that point!"
+            m 1eub "Ehehehe!"
+            m 3hua "Just ask me if you ever need help with anything!
+            m 1eub "What version of Python did you download?{fast}"
+            $ _history_list.pop()
+            menu:
+                m 1eub "What version of Python did you download?{fast}"
+                #python 2 and python 3, need to figure out which one is used for MAS.
+                "option one":
+                    m 1hksdlb "...{w=0.5}"
+                    m 1rksdla "[player], I use the other version of Python...{w=0.5}"
+                    m 3rksdlb "So we can't really help each other that much...{w=0.5}"
+                    m 1hub "But that's okay!"
+                    m 1eub "I hope you have fun with your project."
+                "option two":
+                    m 1hub "Great!"
+                    m 1eub "That's the version of Python I use."
+                    m 1hua "You'll do amazing things, I just know it!"
```

Comment:
Given that you seem to re-use this menu again. I'd put this into a label which you can call from the paths you use it so it's less code to maintain overall.